src/main/java/com/faforever/api/map/MapService.java

```diff
@@ -268,7 +268,7 @@ private void updateMapEntities(MapUploadData progressData) {
       .setDescription(scenarioInfo.get(ScenarioMapInfo.DESCRIPTION).tojstring().replaceAll("<LOC .*?>", ""))
       .setWidth(size.get(1).toint())
       .setHeight(size.get(2).toint())
-      .setHidden(false)
+      .setHidden(true)
```

Comment:
Waaait a second... this was hardcoded all the time!? That would explain some reported weird behavior.
Wouldn't any update on this mapVersion make it hidden again? Like unhide it would make it hidden again? So we are in an endless loop?

We need to distinguish between initial creation and update, don't we?

Reply:
do you want to rename that method?

Reply:
I checked the code now and not just the changed snippet. It looks fine. Just needs the context.